app/services/data_migrations/revert_application_choices_updated_at.rb

```diff
@@ -0,0 +1,26 @@
+module DataMigrations
+  class RevertApplicationChoicesUpdatedAt
+    TIMESTAMP = 20240923111225
+    MANUAL_RUN = true
+
+    def change(limit: nil, provider_ids: [])
+      BatchDelivery.new(relation: choices(limit, provider_ids), stagger_over: 5.hours, batch_size: 1000).each do |next_batch_time, choices|
+        RevertApplicationChoicesUpdatedAtWorker.perform_at(next_batch_time, choices.pluck(:id))
+      end
+    end
+
+    def choices(limit, provider_ids)
+      choices = ApplicationChoice
+        .where(updated_at: Date.new(2024, 9, 3).all_day)
+        .where.not(created_at: Date.new(2024, 9, 3).all_day)
+        .where(current_recruitment_cycle_year: [2024, 2023])
+        .where(status: ApplicationStateChange.states_visible_to_provider)
+
+      if provider_ids.present?
+        choices = choices.where(provider_ids:)
```

Comment:
provider_ids is an array on thie choice table, this means that we can't do `where provider_ids in (1,2)` it needs to match exactly. The value can be [1,2,3] or [1]